src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -270,6 +223,53 @@ impl StaticHeader {
         }
     }
 
+    /// Read the BDA from the device and return 2 SECTORS worth of data, one for each BDA returned
+    /// in the order of layout on disk (location 1, location 2).
+    /// Only the BDA sectors are read up from disk, zero areas are *not* read.
+    fn read<F>(
+        f: &mut F,
+    ) -> (
+        io::Result<[u8; bytes!(static_header_size::SIGBLOCK_SECTORS)]>,
+        io::Result<[u8; bytes!(static_header_size::SIGBLOCK_SECTORS)]>,
+    )
+    where
+        F: Read + Seek,
+    {
+        // Theory of read procedure
```

Comment:
Please remove this comment, too. The topic is covered in the suggested header comment, and a lot of the nomenclature is wrong in this comment.